getting-started/create-project.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create Your First Project
+short_title: Create Project
+---
+
+# Create Your First Project
+
+**Goal:** Create and organize your first Curvenote project so you can start writing and publishing content.  
+
+
+
+## 1. What’s a Project in Curvenote?
+
+A **project** is the home for your articles, figures, datasets, and other content.  
+It also stores your site configuration so you can publish to the web.
```

Comment:
It might be worth expanding on "what is a projects" a little more conceptually. On the previous page you talk about "What you can publish" - single article, multi-article collections, and research outputs. Is a single article a project? (Yes) Are multi-article collections a single project? (More complicated...)